tests/integration/version/version.go

```diff
@@ -0,0 +1,28 @@
+package version
+
+import (
+	"github.com/coreeng/corectl/testdata"
+	"github.com/coreeng/corectl/tests/integration/testconfig"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("version", Ordered, func() {
+	t := GinkgoT()
+	var (
+		corectl *testconfig.CorectlClient
+	)
+
+	BeforeAll(func() {
+		homeDir := t.TempDir()
+		corectl = testconfig.NewCorectlClient(homeDir)
+	})
+
+	Context("version", func() {
+		It("returns sensible defaults", func() {
+			output, err := corectl.Run("version", testdata.DevEnvironment())
```

Comment:
Is the second argument necessary?

I would also propose to add to the command cfg:
```go
		Args:  cobra.ExactArgs(1),
```
Similar to https://github.com/coreeng/corectl/blob/ea5138848c03c56d43c64c7bc2c5212f4468b033/pkg/cmd/tenant/describe/tenant_describe.go#L22